src/index.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable*/
```

Comment:
- [ ] Kindly consider avoiding the disabling of `ESLint` rules as it is designed to assist in writing code that aligns with `industry standards` and `best practices`. Disabling `ESLint` rules can lead to code that may not adhere to these standards, potentially resulting in code quality and maintainability issues. Instead, it is recommended to address any specific `ESLint` rule violations by finding alternative solutions, ensuring that the code remains consistent and of high quality.

Reply:
@Strangeal 
resolved the issue